src/components/propositionalLogic/wiki.tsx

```diff
@@ -20,17 +20,36 @@ class WikiPropositionalLogic extends React.Component {
 				<p>Atomare Formeln können mit den Junktoren
 				&#172; (nicht), &amp; (und), | (oder), &rarr; (Implikation) und &harr; (Äquivalenz).
 				&#172; negiert den Wahrheitswert einer Formel. Das heißt eine wahre Formel wird falsch
-				und eine falsche Formel wird wahr. &#172; ist der
-				einzige Junktor, der nur auf einer statt auf zwei Formeln
-				operiert.
+				und eine falsche Formel wird wahr. Statt &amp; sieht man normalerweise
+				das Zeichen &#8743; und statt | das Zeichen &#8744;. Auf einer gewöhnlichen Tastatur
+				gibt es diese Zeichen jedoch nicht. Daher benutzen wir in Diproche lieber die ersten
+				beiden Zeichen.
+				&#172; ist der einzige Junktor, der nur auf einer statt auf zwei Formeln operiert.
 				Wenn A und B Formeln sind, dann ist A&amp;B wahr genau dann, wenn
 				sowohl A, als auch B wahr ist. Ansonsten ist A&amp;B falsch.
 				A | B ist wahr genau dann, wenn mindestens eine der Formeln
 				A und B wahr ist. Ansonsten ist A | B falsch.
-				A&arr;B ist wahr genau dann, wenn entweder die Voraussetzung A
-				falsch ist oder A und B wahr sind. Ansonsten ist A&arr;B falsch.
+				A&rarr;B ist wahr genau dann, wenn entweder die Voraussetzung A
+				falsch ist oder A und B wahr sind. Ansonsten ist A&rarr;B falsch.
 				A&harr;B ist wahr genau dann, wenn A und B beide falsch oder beide
 				wahr sind. Ansonsten ist A&harr;B falsch.</p>
+				<h2>Klassifikation einige Formeln</h2>
+				<p> Sei F eine Formel mit n Aussagenvariablen. Man kann allen Aussagenvariablen
+					von F auf 2<sup>n</sup> verschiedene Weisen einen Wahrheitswert zuordnen.
+					F heißt nun <b>allgemeingültig</b>, falls jede Wertebelegung F wahr macht.
+					Gibt es mindestens eine Wahrheitswertebelegung, sodass F wahr wird, so heißt
+					F <b>erfüllbar</b>. Gibt es keine solche Wahrheitswertebelegung, dann heißt F
+					<b>unerfüllbar</b>.
+				</p>
+				<h2>Was ist ein Beweis?</h2>
+				Wenn man etwas beweisen muss, so handelt es sich immer um eine Aussage.
+				Wir nennen die zu zeigende Aussasge Z. Eventuell gibt es auch einige Annahmen
+				die man treffen kann. Diese können auch als Aussage gesehen werden. Wir nennen
+				sie A. Dann besteht ein Beweis von Z daraus, Formeln zu finden, sagen wir B,..., Y,
+				sodass A&rarr;B&rarr;...&rarr;Y&rarr;Z eine allgemeingültige Formel ist. Es muss also eine
+				Abfolge von Schlussfolgerungen gefunden werden, die in jedem Fall wahr ist. Streng genommen
+				können in allen Aussagen auch "quantifizierte" Variablen auftauchen. Dies ist aber
```

Comment:
Hier ist quantifizieren für einen totalen Beginner ein komplettes Fremdwort. Du setzt es bereits in Anführungsstrichen, aber ich denke um diese Einführung richtig zu handhaben würde man entweder es wieder erklären (was den Rahmen hier sprengen würde), Stichworte nennen, wo man dazu mehr lesen kann (eventuell in Klammern wie `(Stichworte. Quantoren, All-Quantor, Existenz-Quantor)`) oder man lässt es komplett weg.

Entsprechend würde ich im Nachsatz nicht sagen, dass sie für die Aussagenlogik nicht wichtig sind. Sondern, darauf verweisen, dass diese später behandelt werden. Eine Phrase wie `nicht wichtig`, sollte man in der Lehre vermeiden, auch wenn man es für den jetztigen Moment nur meint. Eventuell besser: `werden (in der X) nicht Betracht` oder `werden wir später behandeln`.

Reply:
Das ist allgemein ein schwieriger Punkt. Unerwähnt wollte ich es nicht lassen, da man sich sonst fragt "Ich muss jetzt was für alle natürlichen Zahlen beweisen, aber davon gibt es unendlich viele und diese Schreibweise "A -> B -> ... -> Y -> Z" heißt doch, dass ein Beweis nur endlich viele Schritte hat." Mit anderen Worten: Man könnte die Idee kriegen, dass ein Beweis durch vollständige Induktion keiner sein kann, da man theoretisch unendlich viele Schritte braucht.

Anführungszeichen gepaart mit dem "das ist hier nicht wichtig" war mein Ansatz zu sagen "Es gibt auch Aussagen mit Variablen, aber sieh mal noch darüber hinweg".  "Werden wir später behandeln" würde ich ungerne nehmen, da das zu sehr nach einem Versprechen klingt, was wir ja erstmal nicht erfüllen. Aktuell haben wir ja nur die Spielwiese Aussagenlogik.
Ich überleg mir mal was für den Teil